app/views/layout.server.view.html

```diff
@@ -40,6 +40,13 @@
 	<!--[if lt IE 9]>
 		<script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
 	<![endif]-->
+
+	{% if user %}
+	<!--Embedding The User Object-->
+	<script type="text/preloaded" name="user">
```

Comment:
@ilanbiala, see below:
* name is not a valid attribute on a script tag.
* name serves as the array key of $preloadProvider constant.